### PR TITLE
Wd arrays scala testing

### PR DESCRIPTION
**Pull request by wjdingman1**
Add Arrays to new testing framework

### CODE REVIEW

File: verification/test-wdls/TestArrays.wdl [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+      }
+
+      call Utilities.GetValidationInputs as GetParams { 
+        input:
+          input_file   = Arrays.chip_well_barcode_params_file,
+          results_path = results_path,
+          truth_path   = truth_path
+      }
+
+      call VerifyArrays.VerifyArrays as VerifyArrays {
```
Comment (nikellepetrillo) [inline verification/test-wdls/TestArrays.wdl:273]: do you need to add the dummy input "done" to force completion before we verify?

Comment (wjdingman1) [reply verification/test-wdls/TestArrays.wdl:273]: good catch!

**Review by nikellepetrillo — APPROVED** [review]

**Comment by wjdingman1** [comment]
@gbggrant can you give this a look over when you have the chance

File: pipelines/broad/arrays/single_sample/Arrays.wdl [hunk 2]
```diff
@@ -495,6 +495,7 @@ workflow Arrays {
     File gtc_file = IlluminaGenotypingArray.gtc
     File red_idat_md5_cloud_path = IlluminaGenotypingArray.red_idat_md5_cloud_path
     File green_idat_md5_cloud_path = IlluminaGenotypingArray.green_idat_md5_cloud_path
+    File output_beadpool_manifest_file = bpm_file
```
Comment (gbggrant) [inline pipelines/broad/arrays/single_sample/Arrays.wdl:498]: super nit:
```suggestion
    File output_bead_pool_manifest_file = bpm_file
```

**Review by gbggrant — APPROVED** [review]
Looks good.

**Pull request merged**